app/views/cars/edit.erb

```diff
@@ -0,0 +1,39 @@
+<div class="container">
+  <% if @errors %>
```

Comment:
If you're sending the ```@car``` instance variable from your controller you don't need to send this ```@errors``` variable. It's storing the @car.errors.full_messages (most likely) so you can just access that from car.